public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -251,29 +233,66 @@ export class PanelInspector extends PureComponent<Props, State> {
   }
 
   renderStatsTab() {
-    const { stats } = this.state;
+    const { last } = this.state;
+    const { request } = last;
+
+    if (!request) {
+      return null;
+    }
+
+    let stats: QueryResultMetaStat[] = [];
+
+    const requestTime = request.endTime ? request.endTime - request.startTime : -1;
+    const processingTime = last.timings?.dataProcessingTime || -1;
```

Comment:
Interesting... never seen this before :)  https://github.com/grafana/grafana/blob/master/public/app/features/dashboard/state/runRequest.ts#L226

I assume this property is not defined in the main model since it is super experimental and we will see what values make sense 👍

Reply:
Added last week ;)